09-schema/01-concepts.md

```diff
@@ -751,16 +751,90 @@ TypeQLDefine query = TypeQL.define(
 [tab:end]
 </div>
 
-## Undefine
-As the name suggests, we use the `undefine` keyword to remove the definition of a type or its association with other types from the schema.
+## Modify an existing schema
+We can modify an existing schema by adding or deleting concepts: entities, relations, attributes and rules.
+
+To add we use `define` and to delete an existing concept we use `undefine`. 
+
+We can't modify concepts in a schema directly. To modify an existing concept in a schema (e.g. rename something) we 
+shall delete the old concept (by using [undefine](#deleting-from-a-schema-with-undefine)) and add the new one (by using 
+[define](#adding-to-a-schema-with-define)).
+
+Don't forget to modify all references accordingly (with deletion and adding).
+
+If there are a lot of changes to be done consider creating a brand-new database to load a new schema from scratch.
```

Comment:
It might be the case that creating a new database would involve also migrating all of the data (could be very painful) - perhaps we could advise that the user make use of Studio as it supports schema editor actions without starting from scratch.

Reply:
Never heard of such an ability/function in Studio.
Most pains in modifying schema are from the need to delete/modify sub-types and data. 
So deleting all data might be a by-product of creating a new DB, but it is often a necessity.
Let's add a note here about adding data from scratch and leave it that way. 
And I want to know more about this Studio schema editor.

Reply:
I think I managed to do both with minimal changes. https://github.com/vaticle/docs/pull/652#:~:text=85020f7-,Fix%20after%20review,-%E2%80%A6